types.go

```diff
@@ -1,33 +1,37 @@
 package smt
 
-import (
-	"encoding/binary"
-	"hash"
-)
+// TODO_DISCUSS_CONSIDERIN_THE_FUTURE:
+// 1. Should we rename all instances of digest to hash?
+// 	> digest is the correct term for the output of a hashing function IIRC
+// 2. Should we introduce a shared interface between SparseMerkleTrie and SparseMerkleSumTrie?
+// 	> Sum() would have to be no-op but could be done
+// 3. Should we rename Commit to FlushToDisk?
+// > No because what if this is an in memory trie?
 
 const (
-	left    = 0
-	sumSize = 8
```

Comment:
Not sure where you moved it but itd now called `sumSizeBits` shouldnt it be `sumSizeBytes` as it represents a uint64 not uint8